my_dwa_planner:

    # forbid lateral movement along the y-axis
    min_vel_y: 0.0
    max_vel_y: 0.0

    # forbid backward movement along the x-axis
    min_vel_x: 0.0
    max_vel_x: 10.0

    # maximum angular acceleration
    acc_lim_theta: 10.0

    # number of uniformly-sampled values of (v,w)
    vx_samples: 30
    vth_samples: 100

    # distance from the center of the robot to place an additional scoring point
    forward_point_distance: 0.05

    # weights of the terms in the cost function
    path_distance_bias: 50.0
    goal_distance_bias: 100.0

    # tolerances for reaching the goal
    xy_goal_tolerance: 0.15
    yaw_goal_tolerance: 6.3
